torch/functional.py

```diff
@@ -535,9 +559,15 @@ def istft(input, n_fft, hop_length=None, win_length=None, window=None,
             centered at time :math:`t \times \text{hop\_length}`.
             (Default: ``True``)
         normalized (bool): Whether the STFT was normalized. (Default: ``False``)
-        onesided (bool): Whether the STFT is onesided. (Default: ``True``)
+        onesided (Optional[bool]): Whether the STFT is onesided.
+            (Default: ``True`` if ``n_fft != fft_size`` in the input size)
```

Comment:
nit: "was" vs. "is" confusion in the descriptions of "normalized" and "onesided", but this was already present in the docs so no need to address it